src/components/LoansByCategory/MFIRecommendations/MfiLoanCard.vue

```diff
@@ -0,0 +1,366 @@
+<template>
+	<div
+		class="tw-flex tw-flex-col"
+		style="width: 100%; height: 100%;"
+		:id="`${loanId}-loan-card`"
+	>
+		<!-- Borrower image w/ location <summary-tag> -->
+		<kv-loading-placeholder
+			v-if="isLoading"
+			class="tw-mb-1 tw-rounded" :style="{width: '100%', height: '168px'}"
+		/>
+		<div
+			v-if="!isLoading"
+			class="tw-relative"
+		>
+			<borrower-image
+				class="
+				tw-relative
+				tw-w-full
+				tw-bg-black
+				tw-rounded
+			"
+				:alt="'photo of ' + borrowerName"
+				:aspect-ratio="3 / 5"
+				:default-image="{ width: 260 }"
+				:hash="imageHash"
+				id="borrower-image"
+			/>
+			<div v-if="countryName">
+				<summary-tag
+					class="tw-absolute tw-bottom-2 tw-left-0 tw-ml-2 tw-text-primary"
+					:city="city"
+					:state="state"
+					:country-name="countryName"
+				>
+					<kv-material-icon
+						class="tw-h-2.5 tw-w-2.5 tw-mr-0.5"
+						:icon="mdiMapMarker"
+					/>
+					{{ formattedLocation }}
+				</summary-tag>
+			</div>
+		</div>
+
+		<!-- Borrower name-->
+		<kv-loading-placeholder
+			v-if="isLoading"
+			class="tw-mb-0.5" :style="{width: 75 + (Math.random() * 15) + '%', height: '1.875rem'}"
+		/>
+
+		<borrower-name
+			v-if="!isLoading"
+			class="tw-mb-1 tw-text-h3"
+			:max-length="50"
+			:name="borrowerName"
+			style="min-height: 3.6rem;"
+		/>
+
+		<!-- LoanUse  -->
+		<kv-loading-paragraph
+			v-if="isLoading"
+			class="tw-mb-1.5 tw-flex-grow" :style="{width: '100%', height: '46px'}"
+		/>
+
+		<div class="flex">
```

Comment:
`tw-flex`?

Reply:
Please fix or remove if activating flex breaks the layout